src/main/resources/interface-specification.yml

```
@@ -836,22 +830,58 @@ components:
         - name
         - purposes
         - kind
+    ClientEntry:
+      description: Models Client details
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+        description:
+          type: string
+        consumer:
```

Note: the returned model should be exactly the one in the read model, unless we need to hide informations
```suggestion
        consumerId:
```